go.mod

```diff
@@ -231,18 +235,21 @@ require (
 	github.com/olekukonko/tablewriter v0.0.5 // indirect
 	github.com/onsi/ginkgo/v2 v2.3.1 // indirect
 	github.com/opencontainers/go-digest v1.0.0 // indirect
-	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198 // indirect
+	github.com/opencontainers/image-spec v1.1.0-rc5 // indirect
+	github.com/opencontainers/runc v1.1.10 // indirect
+	github.com/opencontainers/runtime-spec v1.1.0 // indirect
 	github.com/openshift/assisted-service v0.0.0-20220928142635-a40422bdea61 // indirect
 	github.com/openshift/assisted-service/api v0.0.0 // indirect
 	github.com/openshift/assisted-service/models v0.0.0 // indirect
 	github.com/openshift/cloud-credential-operator v0.0.0-20220316185125-ed0612946f4b // indirect
-	github.com/openshift/cluster-api v0.0.0-20191129101638-b09907ac6668 // indirect
```

Comment:
This is being removed by mod tidy although it is still an indirect dependency in the fork:
https://github.com/openshift/installer-aro/blob/release-4.12-azure/go.mod#L213